test/test_transforms.py

```diff
@@ -28,7 +28,227 @@
 
 GRACE_HOPPER = get_file_path_2(
     os.path.dirname(os.path.abspath(__file__)), 'assets', 'encode_jpeg', 'grace_hopper_517x606.jpg')
+	
 
+def test_random_affine():
```

Comment:
Please write these new tests at the end of the file instead of at the beginning,  it will be easier to find the remaining tests that need to be ported :)